qiskit/providers/ibmq/ibmqsingleprovider.py

```diff
@@ -73,30 +74,34 @@ def _authenticate(self, credentials):
             credentials (Credentials): Quantum Experience or IBMQ credentials.
 
         Returns:
-            IBMQConnector: instance of the IBMQConnector.
+            IBMQClient: instance of the IBMQClient if using the new API.
+            IBMQConnector: instance of the IBMQConnector if using the old API.
         Raises:
             ConnectionError: if the authentication resulted in error.
         """
         # Use an IBMQVersionFinder for finding out the API version.
         proxies = credentials.proxies.get('urls')
+        ntlm_credentials = credentials.proxies.get('ntlm_credentials')
+        auth = None if not ntlm_credentials else HttpNtlmAuth(
+            ntlm_credentials['username'],
+            ntlm_credentials['password']
+        )
```

Comment:
Can you revise this block? `credentials.proxies.get('ntlm_credentials')` will always return `False` - you can get a glimpse of how the original configuration is specified [here](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/qiskit/providers/ibmq/api/utils.py#L171), and trace accordingly (probably using the old api as a reference would help).

We should also take into consideration some subtleties to avoiding `_authenticate()` failing ungracefully (that _both_ username and password are present, etc).